src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -33,6 +34,38 @@ using namespace tvg;
 
 static const char* NoError = "None";
 
+#ifdef THORVG_WG_RASTER_SUPPORT
+    static WGPUInstance instance{};
+    static WGPUAdapter adapter{};
+    static WGPUDevice device{};
+
+    void initWgInstance()
+    {
+        //Init WebGPU
+        if (!instance) instance = wgpuCreateInstance(nullptr);
+
+        // request adapter
+        if (!adapter) {
+            const WGPURequestAdapterOptions requestAdapterOptions { .nextInChain = nullptr, .powerPreference = WGPUPowerPreference_HighPerformance, .forceFallbackAdapter = false };
```

Comment:
@SergeyLebedkin I found the `.compatibleSurface` parameter is not mandatory for browser support. Requiring surface on request adapter part makes browser initialization difficult.

Removed the param, please share your thought if you have any.

Reply:
@tinyjin 
`.compatibleSurface ` parameter is not mandatory for linux/macos. So if in browser its not mandatory, please, remove it